ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -211,6 +211,12 @@ public class ArcticManagementConf {
           .defaultValue(30)
           .withDescription("session timeout in minute");
 
+  public static final ConfigOption<Long> BLOCKER_TIMEOUT =
+      ConfigOptions.key("arctic.ams.blocker.timeout")
```

Comment:
- Remove the prefix `arctic.ams` from the parameter names.
- Add the new configuration into `dist/main/arctic-bin/conf/config.yaml`